src/ansys/fluent/core/launcher/pyfluent_enums.py

```diff
@@ -233,3 +249,35 @@ def _get_standalone_launch_fluent_version(
 
     # 2. the latest ANSYS version from AWP_ROOT environment variables
     return FluentVersion.get_latest_installed()
+
+
+def _get_ui_mode(
+    show_gui: Optional[bool] = None,
+):
+    """Get the graphics driver.
+
+    Parameters
+    ----------
+    show_gui: bool
+        Whether to show the Fluent GUI.
+
+    Returns
+    -------
+    ui_mode: UIMode
+        Fluent GUI mode.
+    """
+    ui_mode = None
+    if show_gui is not None:
+        warnings.warn(
+            "'show_gui' is deprecated, use 'ui_mode' instead",
```

Comment:
```suggestion
            "'show_gui' is deprecated. Use 'ui_mode' instead.",
```
Punctuation should be used in message strings.

Reply:
Done. Thank you.